packages/yoroi-extension/app/containers/profile/LanguageSelectionPage.js

```diff
@@ -48,6 +49,15 @@ export default class LanguageSelectionPage extends Component<InjectedOrGenerated
     }
 
     await this.generated.actions.profile.resetLocale.trigger();
+
+    // Todo: Should be removed with the revamp release
+    if (environment.isNightly() || environment.isDev()) {
+      // Make`revamp` the default theme when it's the first time to install Yoroi.
+      // Do it here as this is the first screen users see.
```

Comment:
Why do it here instead of changing the default theme in `BaseProfileStore`?

Reply:
Nice! I did it